src/components/Episode/Episode.jsx

```diff
@@ -51,16 +48,16 @@ export default function Episode(props){
 				<p>
 					{ synopsis }
 				</p>
-				<a 
-					href={ videoSrc } 
-					rel="noopener" 
-					target="_blank"
-				>
-					<img 
-						src={ thumbSrc } 
-						alt={ thumbDesc } 
-					/>
-				</a>
+				<EpisodePreview 
+					episodeId={ id }
+					index={ index }
+					number={ number }
+					title={ title }
+					thumbnail={ thumbnail }
+					trailer={ trailer }
+					prevTrailer={ prevTrailer }
+					nextTrailer={ nextTrailer }
```

Comment:
mmm, not sure if I like this component receiving props JUST to pass them down to a child component - seems like the sort of thing that `context` would be used for? Let's do that - in the `renderEpisode` util, let's do an `<Episode.Provider value={{ prevTrailer, nextTrailer }}>` so that `<EpisodePreview>` can just do `const { prevTrailer, nextTrailer } = useContext(Episode);` - no need for a big old reducer or anything as it doesn't need any setters - it's just providing context without the need to pass props down multiple levels...

Reply:
might need to create a `context.jsx` and `<EpisodeProvider>` for this? not sure.

Reply:
Created a TrailersProvider which is now passing prevTrailer and nextTrailer objects, which are in turn destructured inside EpisodePreview for their sources.

Reply:
Oops, totally forgot to remove those references 👀